validator/lib/readTileset.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+var fs = require('fs-extra');
+var Promise = require('bluebird');
+
+var fsReadFile = Promise.promisify(fs.readJson);
+
+function readTileset(filePath) {
+    return fsReadFile(filePath)
+        .then(function (jsonObject) {
+            return jsonObject;
+        })
+        .catch(function(error)  {
+            throw error;
+        });
```

Comment:
No need to catch and then throw the error. The error will propagate with the returned promise. Anything that calls `readTileset` would then be responsible for catching the error:

```javascript
readTileset('tileset.json')
    .then(function(json) {

    })
    .catch(function(error) {
        console.log('there was an error')
    });
}
```